README.md

```diff
@@ -164,6 +165,23 @@ We write all the data to AWS S3 buckets:
 - `near-lake-data-testnet` (`eu-central-1` region) for testnet
 - `near-lake-data-mainnet` (`eu-central-1` region) for mainnet
 
+## Custom S3 storage
+
+In case you want to run you own near-lake instance and store data in some S3 compatible storage (Minio or Localstack as example)
```

Comment:
Could you convert `Minio` and `Localstack` words into links to those projects, please?

Reply:
Done